server/fishtest/views.py

```diff
@@ -1408,7 +1408,7 @@ def tests_delete(request):
             validate(runs_schema, run, "run")
         except ValidationError as e:
             message = (
-                f"The run object {request.POST["run-id"]} does not validate: {str(e)}"
+                f"The run object {request.POST['run-id']} does not validate: {str(e)}"
```

Comment:
I wonder if this fixed the issue of the missing print in case of no validation...

Reply:
For me (Python 3.10.8) the first version gives an error
```
(env) fishtest@ip-172-31-50-113:~/fishtest$ pserve ~/fishtest/server/production.ini http_port=6543
Traceback (most recent call last):
  File "/home/fishtest/fishtest/server/env/bin/pserve", line 8, in <module>
    sys.exit(main())
  File "/home/fishtest/fishtest/server/env/lib/python3.10/site-packages/pyramid/scripts/pserve.py", line 30, in main
    return command.run()
  File "/home/fishtest/fishtest/server/env/lib/python3.10/site-packages/pyramid/scripts/pserve.py", line 271, in run
    app = loader.get_wsgi_app(app_name, config_vars)
  File "/home/fishtest/fishtest/server/env/lib/python3.10/site-packages/plaster_pastedeploy/__init__.py", line 111, in get_wsgi_app
    return loadapp(
  File "/home/fishtest/fishtest/server/env/lib/python3.10/site-packages/paste/deploy/loadwsgi.py", line 248, in loadapp
    return loadobj(APP, uri, name=name, **kw)
  File "/home/fishtest/fishtest/server/env/lib/python3.10/site-packages/paste/deploy/loadwsgi.py", line 273, in loadobj
    return context.create()
  File "/home/fishtest/fishtest/server/env/lib/python3.10/site-packages/paste/deploy/loadwsgi.py", line 741, in create
    return self.object_type.invoke(self)
  File "/home/fishtest/fishtest/server/env/lib/python3.10/site-packages/paste/deploy/loadwsgi.py", line 138, in invoke
    return fix_call(context.object, context.global_conf, **context.local_conf)
  File "/home/fishtest/fishtest/server/env/lib/python3.10/site-packages/paste/deploy/util.py", line 61, in fix_call
    val = callable(*args, **kw)
  File "/home/fishtest/fishtest/server/fishtest/__init__.py", line 180, in main
    config.scan()
  File "/home/fishtest/fishtest/server/env/lib/python3.10/site-packages/pyramid/config/__init__.py", line 880, in scan
    scanner.scan(
  File "/home/fishtest/fishtest/server/env/lib/python3.10/site-packages/venusian/__init__.py", line 220, in scan
    __import__(modname)
  File "/home/fishtest/fishtest/server/fishtest/views.py", line 1411
    f"The run object {request.POST["run-id"]} does not validate: {str(e)}"
                                    ^^^
SyntaxError: f-string: unmatched '['
```
That's why I changed it to the second version.

Since this is only an error message I do not see how this can matter....

Reply:
Of course, is an error, that code tried (lazily) to fix the previous error so that code never worked.